assets/i18n/diq.json

```diff
@@ -73,7 +73,7 @@
     "I18N_INTERACTIONS_SET_INPUT_EMPTY_SET": "kHer satır rê jew wesiqe dek)",
     "I18N_INTERACTIONS_SET_INPUT_NO_ANSWER": "cewab nêdeya.",
     "I18N_INTERACTIONS_SET_INPUT_SUBMIT": "Bırışe",
-    "I18N_LANGUAGE_FOOTER_VIEW_IN": "Oppa'y ena zıwan dı bıasnê:",
+    "I18N_LANGUAGE_FOOTER_VIEW_IN": "Oppai'y ena zıwan dı bımotné:",
```

Comment:
it seems like a mistake to me. Is it possible to confirm with someone who knows this language? the translatewiki fix will need to wait for a future release right?